templates/pipeline

```diff
@@ -0,0 +1,50 @@
+## THIS FILE WAS AUTOMATICALLY GENERATED {{  }} ##
```

Comment:
Why do we have this file as well as garden_ai/templates/pipeline?

Reply:
we shouldn't, just slipped by me -- this is where I'd assumed jinja/poetry would look for templates but they needed to be down a level